src/components/CommentBox.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react"
+import { getComments } from "../api"
+import CommentCard from "./CommentCard"
+
+export default function CommentBox (props){
+    const [comments, setComments] = useState([])
```

Comment:
Ideally this component will have it's own loading state, as it is possible the comments could take longer than the rest of the article to load.